DeploymentGuide.md

```diff
@@ -12,25 +12,36 @@ The Management Server can be deployed locally or to the cloud. We have setup a o
 
 The click to deploy method creates a new app based on a template from github. Heroku will read the configurations already created there and generate a new app for you, with almost no interaction required. Simply follow these steps:
 
+##### Deploy Your App
```

Comment:
Part of Greg's suggestions is also to have step numbers, which I think is a good one as they would outline the sub-steps here.

Step 1: Create an Instance via Click-to-Deploy.
Step 2: Upgrade Your App to Hobby Level
Step 3: Configuring Your App's Settings